mock/py/mockbuild/plugins/lvm_root.py

```diff
@@ -382,6 +382,21 @@ def hook_remove_snapshot(self, name):
         lvm_do(['lvremove', '-f', self.vg_name + '/' + lv_name])
         self.buildroot.root_log.info("deleted {name} snapshot".format(name=name))
 
+    def hook_postupdate(self):
+        name = self.get_current_snapshot()
+        if re.match(r".*_updated_[\-\_0-9]{16}$", name):
+            name = re.sub(r"_updated_[\-\_0-9]{16}$", "", name)
+
+        timestamp = time.localtime(time.time())
+        name = name + "_updated_{y}-{mo:02d}-{d:02d}_{h:02d}-{m:02d}".format(
+            y=timestamp.tm_year, mo=timestamp.tm_mon, d=timestamp.tm_mday,
+            h=timestamp.tm_hour, m=timestamp.tm_min
+        )
```

Comment:
The name is pretty complicated.  I think it would be enough to have just the numeric `time.time()` output there, but no problem here .. just saying.

Reply:
Just checking, are the "updated" snapshot deleted with --scrub?

Reply:
Yes, they are.